modules/wayprompt/meta.nix

```
@@ -0,0 +1,5 @@
+{ lib, ... }:
+{
+  name = "wayprompt";
```

won't do anything until https://github.com/danth/stylix/pull/1083/ is merged, but would be nice to already have it set
```suggestion
  name = "wayprompt";
  homepage = "https://git.sr.ht/~leon_plickat/wayprompt";
```